clients/drcachesim/simulator/cache_simulator.cpp

```diff
@@ -578,26 +578,31 @@ cache_simulator_t::print_results()
         print_core(i);
         if (thread_ever_counts_[i] > 0) {
             if (l1_icaches_[i] != l1_dcaches_[i]) {
-                std::cerr << "  L1I stats:" << std::endl;
+                std::cerr << "  " << l1_icaches_[i]->get_name()
+                          << " stats: " << l1_icaches_[i]->get_description() << std::endl;
```

Comment:
So it would be like:
```
L1D stats: size=n, assoc=n, block=n, FIFO, coherent
  Hits:
```

It may be better as:
```
L1D (size=n, assoc=n, block=n, FIFO, coherent) stats:
  Hits:
```

Reply:
You're right, that is better.  Just wish it didn't affect so many templates...
Oh, well, right is right.